### DISCUSSION TITLE
Add an option to sync videos only

### DISCUSSION THREAD

**Discussion by chisonne** [gilesknap/gphotos-sync#391] [Ideas]
It would be great to have an option to sync videos only skip photos.
An option similar too --skip-video but for photos.

Videos can take up a lot of space in the Google Photos storage. By syncing videos only users could backup videos in order to remove them from Google Photos to save storage space.

**Comment by gilesknap** [comment]
Thanks for the feature request. I'm not going to implement this because video backup is appalling due to long outstanding issues with Google's API. See #119 for details.
